### DISCUSSION TITLE
Can we automatically set ` Org User Self Manager` on account creation ?

### DISCUSSION THREAD

**Discussion by Yberion** [zitadel/zitadel#7402] [Q&A] [answered]
Hello,

Is there a way to automatically set the org role  `Org User Self Manager` (`ORG_USER_SELF_MANAGER`) on account creation ?

I checked `Actions` docs but it seems not to be possible, unless I didn't get it.

It should be something similar to project grants (https://zitadel.com/docs/apis/actions/internal-authentication#parameters-of-post-creation), but at the org level.

**Comment by hifabienne** [comment]
I think at the moment we are only able to set user grants but not manager roles (internal roles)
@peintnermax @stebenz Can you confirm? Or tell how to achieve that?

**Reply by stebenz** [reply, marked as answer]
@Yberion @hifabienne 
Correct, currently it is only possible to add user grants after the creation.
To achieve that we would need to add the memberships to the action similar to the usergrants.

**Reply by Yberion** [reply]
@stebenz @hifabienne Awesome, thanks for the answer. Should I make a ticket or feature request for this?

**Reply by hifabienne** [reply]
yes that would be great
